feature/selector.py

```diff
@@ -568,8 +644,10 @@ def calculate_statistics(scores: pd.DataFrame,
     ----------
     scores:  pd.DataFrame
         Data frame with scores for each feature (index) and selector (columns).
+        Each feature could have multiple rows from different cross-validation folds.
```

Comment:
This is a comment for the calc_stat() function. It is missing the Return Type

Reply:
Added in this [commit](https://github.com/fidelity/selective/pull/3/commits/281c4f3a95d7764ccd3ea9fbf4a785fe820940b9).